lib/board.rb

```diff
@@ -0,0 +1,51 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+
+    def reset!
+      @cells = Array.new(9, " ")
+    end
+
+    def initialize
+      reset!
+    end
+
```

Comment:
Good use of your `#reset!` method to keep your code DRY.  Conventionally, the `#initialize` method should be the first method listed in a class, so that anyone collaborating or looking at your code can immediately see what happens upon instantiation.